packages/rocketchat-lib/server/models/Subscriptions.coffee

```diff
@@ -19,19 +19,28 @@ class ModelSubscriptions extends RocketChat.models._Base
 		this.cache.ensureIndex('rid', 'array')
 		this.cache.ensureIndex('u._id', 'array')
 		this.cache.ensureIndex(['rid', 'u._id'], 'unique')
+		this.cache.ensureIndex(['name', 'u._id'], 'unique')
```

Comment:
@graywolf336 That is ok. The room name is unique, so room name + user id will be unique for subscriptions